web/modules/custom/dpl_opening_hours/src/Controller/OpeningHoursEditorController.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Drupal\dpl_opening_hours\Controller;
+
+use Drupal\Core\Access\AccessResult;
+use Drupal\Core\Controller\ControllerBase;
+
+/**
+ * Defines OpeningHoursController class.
+ */
+class OpeningHoursEditorController extends ControllerBase {
+
+  /**
+   * Display the opening hours app.
+   *
+   * @return mixed[]
+   *   The app render array.
+   */
+  public function content() : array {
+    return [
+      '#theme' => 'dpl_react_app',
+      '#name' => 'opening-hours-editor',
+    ];
+  }
+
+  /**
+   * Check access for a specific node.
+   */
+  public function access(int $node) : AccessResult {
+    $nodeStorage = $this->entityTypeManager()->getStorage('node');
+    $nodeEntity = $nodeStorage->load($node);
+
+    if (!$nodeEntity) {
```

Comment:
```suggestion
    if (!($nodeEntity instanceof Node)) {
```

That way, PHPStorm and other editors understand what `getType()` means below :)

Reply:
You could also consider adding the check below here:

```
    if (!($nodeEntity instanceof Node) || $nodeEntity->getType() !== 'branch') {
```

Reply:
Improved in [ccad6be](https://github.com/danskernesdigitalebibliotek/dpl-cms/pull/898/commits/ccad6bea9929c813a9f3c8cf8754792e8e3d170a).